tests/common.go

```diff
@@ -1583,6 +1585,31 @@ func ScheduleApplications(testname string, errChan ...*chan error) []*scheduler.
 	return contexts
 }
 
+// ScheduleApplications schedules *the* applications and returns the scheduler.Contexts for each app (corresponds to given namespace). NOTE: does not wait for applications
+func ScheduleApplicationsOnNamespace(nameSpace string, testname string, errChan ...*chan error) []*scheduler.Context {
```

Comment:
Not sure why this was needed

Reply:
Your taskname itself will be present in the namespace

Reply:
If ScheduleApplicationsOnNamespace is needed, please make sure nameSpace (ideally namespace) does not exceed 63 characters.

Reply:
we need separate method if we want to deploy application on custom namespace provided by user ,the default ScheduleApplications will create a new namespace itself by appending taskname and timestamp , For testcase requirement we need to delete and re-create the namespaces with same names.
The taskname will be overridden by namespace in ScheduleApplicationsOnNamespace.